crates/api_models/src/admin.rs

```diff
@@ -93,10 +93,33 @@ pub struct MerchantAccountCreate {
     #[schema(value_type = Option<Object>,example = json!({"type": "single", "data": "signifyd"}))]
     pub frm_routing_algorithm: Option<serde_json::Value>,
 
-    /// The id of the organization to which the merchant belongs to
+    /// The id of the organization to which the merchant belongs to, if not passed an organization is created
     pub organization_id: Option<String>,
 }
 
+#[cfg(feature = "v2")]
+#[derive(Clone, Debug, Deserialize, ToSchema, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct MerchantAccountCreateV2 {
+    /// The identifier for the Merchant Account
+    #[schema(max_length = 64, example = "y3oqhf46pyzuxjbcn2giaqnb44")]
+    pub id: String,
```

Comment:
If we have both `id` and `merchant_id`, we should take merchant_id as input, and we will generate the `id` right?

Reply:
Yes, you are right. We need to discuss about how ids can be referenced in other tables. Whether it is `merchant_id` or `id`. Will keep this thread updated with the information

Reply:
The id for merchant account will be generated from `merchant_name`